// Copyright 2016, Xuanting Cai
// Email: caixuanting@gmail.com

syntax = "proto2";

package strategy_runner;

enum StrategyMessageType {
	ORDER = 1;
	DATA = 2;
}

message StrategyMessage {
	optional string data_id = 1;
	optional string strategy_name = 2;
	optional StrategyMessageType type = 3;
	optional string timestamp = 4;
	optional string timestamp_from_signal = 5;
	optional string timestamp_from_data = 6;
}